inputFiles/poromechanicsFractures/SlipPermeability_embeddedFrac.xml

```diff
@@ -12,7 +12,7 @@
       solidSolverName="matrixMechSolver"
       logLevel="1">
       <NonlinearSolverParameters
-        newtonTol="1.0e-5"
+        newtonTol="1.0e-1"
```

Comment:
this seems really low, could you please comment here?

Reply:
@paveltomin Just try to make sure this case can be running and generate reasonable result. Cannot pin down the reason for the regression of the linear solver performance.